packages/containers/src/GuidedTour/GuidedTour.sagas.test.js

```diff
@@ -0,0 +1,24 @@
+import { show, hide } from './GuidedTour.sagas';
+import Connected from './GuidedTour.connect';
+
+describe('Guided Tour sagas', () => {
+	it('should show the Guided Tour', () => {
```

Comment:
Just a thought, i really give more value to tests using redux-saga-tester

Reply:
I will pass for that time :)